base/applications/osk/settings.c

```diff
@@ -90,6 +97,34 @@ BOOL LoadDataFromRegistry()
 
     /* Load the sound on click value event */
     Globals.bSoundClick = (dwSoundOnClick != 0);
+
+    /* Query the key */
+    lResult = RegQueryValueExW(hKey,
+                               L"DialogPlacement",
```

Comment:
Are you making this up or is it supposed to be Windows-compatible?
I see WindowLeft/WindowTop/WindowWidth/WindowHeight on Win10.

Reply:
Currently it's a make up. Undoubtedly I'll make it Windows compatible later once the code is ready.

Reply:
I don't know whether it makes sense to be compatible with these precise values.
Vista+ OSK (not Win2k3 and less versions!) use these WindowLeft/Top/Width/Height, because the OSK window can be proportionally resized (the buttons inside rescale), while <=Win2k3 and ROS one does not.

Reply:
I guess my main point here was that using separate values would get rid of the printf/scanf stuff and result in cleaner code. If we can be more Windows-compatible (whatever version) at the same time, that seems like the right thing to do.

Reply:
I actually agree with you; and we would just need the *Left andd *Top variables, while ignoring the Width and Height ones.